src/launchpad/NFTLaunchpadCommon.sol

```diff
@@ -7,9 +7,7 @@ import { INFTLaunchpad } from "../interfaces/launchpad/INFTLaunchpad.sol";
 
 abstract contract NFTLaunchpadCommon is IERC165, INFTLaunchpad {
   /// @dev Returns whether the contract supports the NFT launchpad interface.
-  function supportsInterface(
-    bytes4 interfaceId
-  ) public view virtual returns (bool) {
-    return interfaceId == type(INFTLaunchpad).interfaceId;
+  function supportsInterface(bytes4 interfaceId) public view virtual returns (bool) {
+    return interfaceId == type(INFTLaunchpad).interfaceId || interfaceId == type(IERC165).interfaceId;
```

Comment:
```suggestion
    return interfaceId == type(INFTLaunchpad).interfaceId || interfaceId == type(IERC165).interfaceId || interfaceId == super.supportsInterface(interfaceId);
```

Reply:
`super.supportInterface` releases the boolean not byte4 interface id